<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>140</y>
     <width>81</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>نوع العميل </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>نوع العضويه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>رقم الهاتف</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>رقم الموبايل</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>الوظيفه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>العنوان</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>المدينه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>المنطقه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>رقم العماره</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>الطابق</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>اﻹيميل</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>الشركه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>تليفون العمل</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>عنوان العمل</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>حاله العميل</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>ملاحظات</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>561</width>
     <height>523</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="CMB_custGroup"/>
    </item>
    <item>
     <widget class="QComboBox" name="CMB_loyalityType"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lE_phone"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lE_mobile"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_job"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_address"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_city"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_district"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_building"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_floor"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_email"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_company"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_workPhone"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_workAddress"/>
    </item>
    <item>
     <widget class="QComboBox" name="CMB_status"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_notes"/>
    </item>
   </layout>
   <zorder>CMB_loyalityType</zorder>
   <zorder>lE_phone</zorder>
   <zorder>LE_job</zorder>
   <zorder>lE_mobile</zorder>
   <zorder>LE_address</zorder>
   <zorder>LE_city</zorder>
   <zorder>LE_district</zorder>
   <zorder>LE_building</zorder>
   <zorder>LE_floor</zorder>
   <zorder>LE_email</zorder>
   <zorder>LE_company</zorder>
   <zorder>LE_workPhone</zorder>
   <zorder>LE_workAddress</zorder>
   <zorder>CMB_custGroup</zorder>
   <zorder>CMB_status</zorder>
   <zorder>LE_notes</zorder>
  </widget>
  <widget class="QPushButton" name="BTN_modifyCustomer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>تعديل عميل عضويه</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>اسم العميل</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_custName">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>50</y>
     <width>551</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LB_custID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cust id </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
